dateutil/parser/__init__.py

```diff
@@ -3,5 +3,8 @@
 from ._parser import DEFAULTPARSER, DEFAULTTZPARSER
 from ._parser import InvalidDateError, InvalidDatetimeError, InvalidTimeError
 
+from .isoparser import Isoparser, isoparse
+
 __all__ = ['parse', 'parser', 'parserinfo',
+           'isoparse', 'Isoparser',
```

Comment:
ISOParser?

Reply:
I looked into this particularly for the question of how to handle CapWords for TZ and it seems like the consensus is that you don't capitalize all the letters of acronyms, just treat them as any other word (so `IsoParser` instead of `ISOParser`, `TzUtc` instead of `TZUTC`).

That said, despite the fact that Python is pretty weirdly inconsistent on this point, when I actually read PEP 8 recently, they pretty strongly discourage changing an established library style just to be consistent with PEP 8, so I'm thinking we should just embrace `isoparser`.